### DISCUSSION TITLE
Unable to locate package vhs

### DISCUSSION THREAD

**Discussion by JobsKits** [charmbracelet/vhs#341] [Q&A]
root@Ubuntu:~# sudo apt install vhs ffmpeg

Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package vhs

root@Ubuntu:~#

**Comment by bashbunni** [comment]
Hey @295060456 to confirm, did you run the following before trying to install vhs?
```bash
sudo mkdir -p /etc/apt/keyrings
curl -fsSL https://repo.charm.sh/apt/gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/charm.gpg
echo "deb [signed-by=/etc/apt/keyrings/charm.gpg] https://repo.charm.sh/apt/ * *" | sudo tee /etc/apt/sources.list.d/charm.list
```

(*grabbed from [README](https://github.com/charmbracelet/vhs#installation)*)

**Comment by bashbunni** [comment]
Hey @295060456 let us know if you're still having this issue after trying my previous suggestion.

**Comment by kalensk** [comment]
@bashbunni 

I am running into this issue. Can you confirm `curl -fsSL https://repo.charm.sh/apt/gpg.key` that link is valid and present?

**Reply by caarlos0** [reply]
@kalensk should be yes
